Expensify/k2-extension
# Fix repeated fetching in the API

I was finding a few problems recently that I've been fixing.

1. I noticed that the network tab kept having (what seemed) infinite numbers of requests over and over
2. The dashboard would flash an rerender a lot (especially when cmd+click on items)

This PR should fix both of those problems.

# Tests
1. Open the dashboard and watch the network tab for a few minutes. There should only be a dozen or so requests every minute when data gets re-fetched
2. Hold down command and click on several items at the top of the dashboard
3. Verify the page doesn't flash
